ebos/eclproblem.hh

```diff
@@ -2669,11 +2669,10 @@ private:
         referencePorosity_[/*timeIdx=*/0].resize(numDof);
 
         const auto& fp = eclState.fieldProps();
-        const std::vector<double> porvData = fp.porv(true);
+        const std::vector<double> porvData = fp.porv(false);
         const std::vector<int> actnumData = fp.actnum();
         for (size_t dofIdx = 0; dofIdx < numDof; ++ dofIdx) {
-            unsigned cartElemIdx = vanguard.cartesianIndex(dofIdx);
-            Scalar poreVolume = porvData[cartElemIdx];
+            Scalar poreVolume = porvData[dofIdx];
```

Comment:
What does `FieldProps::porv(false)` return in a parallel simulation?  Does it return only those values that are local to the process or does it return a vector of values that is sized according to the number of active cells in the entire model?

Reply:
@joakim-hove This will break flow if called on a distributed grid. As I tried to point out in the bug report #3081 fp.porv(false) returns wrong results. Only the approach taken here (before your change) yields correct results. (Actually I tried to explain that what was done here originally needs to be done there for the argument false).

Your change (without fixing ParallelEclipseState::porv) will have the following effect.
- Each process will have a porv array for the active cells of the undistributed grid.

Are you sure that this is what we want?

Reply:
Ehhh - I will close this now and try to digest what you write. Thanks.